### DISCUSSION TITLE
Move order in tool settings / side panel

### DISCUSSION THREAD

**Discussion by latipun7** [flameshot-org/flameshot#2397] [Q&A] [answered]
How to move layer order of the tools in tool settings / side panel?

If this is not yet available, I'd like to see this feature.
Thank you.

**Comment by mmahmoudian** [comment, marked as answer]
It is available if you build it yourself or use AUR if you are Arch user. Draw multiple things, then open the side bar (for example by pressing <kbd>Space</kbd>) and then select the item you want and use the arrow buttons in the sidebar to move them one layer at a time. It will be released in the next version. Building from source is actually very easy, but alternatively you can download nightly builds from here:

https://flameshot.org/docs/installation/development-build/

**Reply by latipun7** [reply]
I see, so that is a new feature then. Thank you

**Reply by mmahmoudian** [reply]
Yes, this is the Pull Request that added this feature:

https://github.com/flameshot-org/flameshot/pull/2108
